### PR TITLE
Code cleanup & RabbitMQ addition

### PR DESCRIPTION
**Pull request by MilanBrock**

### CODE REVIEW

**Review by ZakariaICT — DISMISSED** [review]
Nice Work!

File: src/main/java/com/babymonitoring/simulationplayer/Simulation.java [hunk 1]
```diff
@@ -142,8 +142,9 @@ public void actionPerformed(ActionEvent e) {
                     series.add(xCoord, yCoord); // Series.add( X-as, Y-as )
                     //---------------------
 
-                    //WEBSOCKET
+                    //----- WEBSOCKET -----
                     //controller.SendCoords(new CoordsMessage(userId, xCoord, yCoord));
+                    //---------------------
```
Comment (Popodil) [inline src/main/java/com/babymonitoring/simulationplayer/Simulation.java:147]: Still necessary?

**Review by Fontie — APPROVED** [review]
**Thumb up**

File: src/main/java/com/babymonitoring/simulationplayer/Simulation.java [hunk 2]
```diff
@@ -218,11 +208,13 @@ public void startProductionSimulation(boolean vMother, boolean vUterus, boolean
...
 
-
+        //Just here if we need it
+        /*
         Object[] qUt = (Object[]) tempSimResults[1];
         double[] qUtT = (double[]) qUt[0];
         double[] qUtV = (double[]) qUt[1];
-        //FHRResult fhrResult = new FHRResult(qUtT, qUtV);
+        FHRResult fhrResult = new FHRResult(qUtT, qUtV);
+        */
```
Comment (Popodil) [inline src/main/java/com/babymonitoring/simulationplayer/Simulation.java:217]: Still necessary?

**Review by Popodil — APPROVED** [review]

**Pull request merged**